### PR TITLE
[14.0][ADD] shopinvader_sale_exception

### PR DESCRIPTION
**Pull request by ivantodorovich**

### CODE REVIEW

File: shopinvader_sale_exception/services/cart.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+class CartService(Component):
+    _inherit = "shopinvader.cart.service"
+
+    def check_exceptions(self, **params):
+        """Run sale exception checks"""
+        cart = self._get(create_if_not_found=False)
+        if not cart:
+            return {}
+        cart.detect_exceptions()
+        return self._to_json(cart)
```
Comment (simahawk) [inline shopinvader_sale_exception/services/cart.py]: do we really have to return the whole cart?
IMO we should return only sale_exceptions here. Not a blocker tho.

Comment (ivantodorovich) [reply shopinvader_sale_exception/services/cart.py]: It's just to be consistent with all the other cart methods, they all return the whole cart data

Comment (simahawk) [reply shopinvader_sale_exception/services/cart.py]: My POV:
1. this endpoint does not alter the cart, right?
2. this endpoint is called explicitly only to get this info, right?

Comment (ivantodorovich) [reply shopinvader_sale_exception/services/cart.py]: True. I've just pushed the changes 👍🏻 thanks!

**Comment by sebastienbeau** [comment]
@simahawk can you do a last review, and I will merge it ?

**Pull request closed without merging**